src/AspNet.Security.OAuth.Validation/Events/AuthenticationFailedContext.cs

```diff
@@ -2,7 +2,7 @@
 using Microsoft.AspNetCore.Http;
 using System;
 
-namespace AspNet.Security.OAuth.Validation.Events
```

Comment:
FYI, this is the pattern used by the ASP.NET team (and reused in ASOS): no `.Events` suffix for the namespace but all the events classes are stored in an `Events` folder.